### PR TITLE
doc: update "What is ACRN" introduction

### PR DESCRIPTION
**Pull request by gvancuts**
Refresh and update the "What is ACRN" document to reflect the latest changes.
Most notably, the scenario definitions have changed.

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

File: doc/introduction/index.rst [hunk 1]
```diff
@@ -65,7 +65,7 @@ VM. The service VM can access hardware resources directly by running
 native drivers and it provides device sharing services to the user VMs
 through the Device Model.  Currently, the service VM is based on Linux,
 but it can also use other operating systems as long as the ACRN Device
-Model is ported into it. A user VM can be Clear Linux*, Android*,
+Model is ported into it. A user VM can be Clear Linux*, Ubuntu*, Android*,
 Windows* or VxWorks*.  There is one special user VM, called a
 post-launched Real-Time VM (RTVM), designed to run a hard real-time OS,
 such as VxWorks*, or Xenomai*. Because of its real-time capability, RTVM
```
Comment (dbkinder) [inline doc/introduction/index.rst:71]: let's add Zephyr* to this list of RTOSs

File: doc/introduction/index.rst [hunk 2]
```diff
@@ -103,10 +103,10 @@ Experience (IVE) solutions.
 As a reference implementation, ACRN provides the basis for embedded
 hypervisor vendors to build solutions with a reference I/O mediation
 solution.  In this scenario, an automotive SDC system consists of the
-Instrument Cluster (IC) system in VM1, the In-Vehicle Infotainment (IVI)
-system in VM2, and one or more Rear Seat Entertainment (RSE) systems in
-VM3. Each system is running as an isolated Virtual Machine (VM) for
-overall system safety considerations.
+Instrument Cluster (IC) system running in the Service VM, the In-Vehicle
```
Comment (dbkinder) [inline doc/introduction/index.rst:106]: change to ``the Service VM and the...``

File: doc/introduction/index.rst [hunk 3]
```diff
@@ -169,49 +163,45 @@ Industrial Workload Consolidation
...
+- The ACRN hypervisor can create VMs that run different OSes:
 
-  - A safety VM such as Zephyr*,
-  - a service VM such as Clear Linux*,
-  - a Human Machine Interface (HMI) application OS such as Windows*, and
+  - a Service VM such as Ubuntu*,
+  - a Human Machine Interface (HMI) application OS such as Windows*,
+  - an Artifical Intelligence (AI) application on Linux*,
+  - a Kata Container application, and
   - a real-time control OS such as VxWorks or RT-Linux*.
```
Comment (dbkinder) [inline doc/introduction/index.rst:203]: add Zephyr

File: doc/introduction/index.rst [hunk 4]
```diff
@@ -275,73 +257,61 @@ most one Safety VM and it is pre-launched.)
...
-======================================
+Industry scenario
+=================
 
 In this Industry scenario, the Service VM provides device sharing capability for
-a Windows-based HMI User VM. The other two post-launched User VMs
-support either hard or soft Real-time OS applications.
+a Windows-based HMI User VM. One post-launched User VM can run a Kata Container
+application. Another User VM supports either hard or soft Real-time OS
+applications. And up to 5 additional post-launched User VMs support functions
```
Comment (dbkinder) [inline doc/introduction/index.rst:282]: delete ``And`` and start the sentence with ``Up to five additional ...``

File: doc/introduction/index.rst [hunk 5]
```diff
@@ -481,20 +451,27 @@ ACRN hypervisor is a Type 1 hypervisor, running directly on bare-metal
...
-:numref:`ACRN-architecture` shows the ACRN hypervisor architecture, with
-the automotive example IC VM and service VM together. The Service VM
-owns most of the devices including the platform devices, and
-provides I/O mediation. Some of the PCIe devices may be passed through
-to the User OSes via the VM configuration. The Service VM runs the IC
-applications and hypervisor-specific applications together, such as the
-ACRN device model, and ACRN VM manager.
+:numref:`V2-hl-arch` shows the ACRN hypervisor architecture, with
+all types of Virtual Machines (VMs) represented:
+- Pre-launched User VM (Safety/RTVM)
```
Comment (dbkinder) [inline doc/introduction/index.rst:463]: add a blank line before the first item of a list

File: doc/introduction/index.rst (hunk 2 above)
Comment (dbkinder) [reply doc/introduction/index.rst:106]: Also change the ``:width: 400px`` to be ``:width: 600px`` in the referenced image (line 97)

File: doc/introduction/index.rst [hunk 6]
```diff
@@ -169,49 +163,45 @@ Industrial Workload Consolidation
    ACRN Industrial Workload Consolidation scenario
 
 Supporting Workload consolidation for industrial applications is even
-more challenging. The ACRN hypervisor needs to run both safety-critical
-and non-safety workloads with no interference, increase security
-functions that safeguard the system, run hard real-time sensitive
-workloads together with general computing workloads, and conduct data
-analytics for timely actions and predictive maintenance.
+more challenging. The ACRN hypervisor needs to run different workloads with no
```
Comment (dbkinder) [inline doc/introduction/index.rst:166]: change the ``:width: 400px`` to ``:width: 600px`` for the image on line 159 as well

**Comment by gvancuts** [comment]
Updated and force-pushed. Thanks!

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br><br>No New Function Declaration/Definition Mismatch<br>

**Pull request merged**